README.rst

```diff
@@ -941,6 +941,15 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_last`` (``bool``): whether ``null`` should be the last type; defaults to
```

Comment:
not good idea IMO, as then one would `first_types=null` and `last_types=null` :D
also, I wouldn't go with that big granularity

what about:
- sort_types: bool (sort all types alphabetically)
- null: [opt] `first|last`
?